NotificationService/RLTNotificationServiceConfigurator.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  RLTNotificationServiceConfigurator.swift
+//  RealifeTech-SDK
+//
+//  Created by YOU-HSUAN YU on 2022/6/27.
+//
+
+import UserNotifications
+
+public class RLTNotificationServiceConfigurator {
```

Comment:
If I understand correctly main purpose of this class is to track notifications.  Maybe something like `RLTNotificationsTracker` would be more suitable?